android/src/main/java/com/facebook/reactnative/androidsdk/Utility.java

```diff
@@ -100,6 +101,61 @@ public static WritableMap accessTokenToReactMap(AccessToken accessToken) {
         return map;
     }
 
+    public static WritableMap profileToReactMap(Profile profile) {
```

Comment:
If you could, just in case would be nice to have some simple utility to putNull or value into the WritableMap to avoid repeating yourself that much in this case as you're having too many `if/else` similarly here.